### PR TITLE
Grap 33

### PR DESCRIPTION
**Pull request by svyatoslavkk**
Add 404 interface and routing for wrong url

### CODE REVIEW

File: src/pages/ErrorPage/styles.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
+import styled, { keyframes } from 'styled-components';
+
```
Comment (ppmarkek) [inline src/pages/ErrorPage/styles.ts:2]: Use mui styled and mui components

File: src/pages/ErrorPage/styles.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
...ZGIiB3aWR0aD0iMSIgaGVpZ2h0PSIxIi8+PHJlY3QgeD0iNTgiIHk9IjgiIGZpbGw9IiNGRkZGRkYiIHdpZHRoPSIxIiBoZWlnaHQ9IjEiLz48cmVjdCB4PSIxNDciIHk9IjMzIiBmaWxsPSIjRkZGRkZGIiB3aWR0aD0iMiIgaGVpZ2h0PSIyIi8+PHJlY3QgeD0iOTEiIHk9IjQzIiBmaWxsPSIjRkZGRkZGIiB3aWR0aD0iMSIgaGVpZ2h0PSIxIi8+PHJlY3QgeD0iMTY5IiB5PSIyOSIgZmlsbD0iI0ZGRkZGRiIgd2lkdGg9IjEiIGhlaWdodD0iMSIvPjxyZWN0IHg9IjE4MiIgeT0iMTkiIGZpbGw9IiNGRkZGRkYiIHdpZHRoPSIxIiBoZWlnaHQ9IjEiLz48cmVjdCB4PSIxNjEiIHk9IjU5IiBmaWxsPSIjRkZGRkZGIiB3aWR0aD0iMSIgaGVpZ2h0PSIxIi8+PHJlY3QgeD0iMTM4IiB5PSI5NSIgZmlsbD0iI0ZGRkZGRiIgd2lkdGg9IjEiIGhlaWdodD0iMSIvPjxyZWN0IHg9IjE5OSIgeT0iNzEiIGZpbGw9IiNGRkZGRkYiIHdpZHRoPSIzIiBoZWlnaHQ9IjMiLz48cmVjdCB4PSIyMTMiIHk9IjE1MyIgZmlsbD0iI0ZGRkZGRiIgd2lkdGg9IjIiIGhlaWdodD0iMiIvPjxyZWN0IHg9IjEyOCIgeT0iMTYzIiBmaWxsPSIjRkZGRkZGIiB3aWR0aD0iMSIgaGVpZ2h0PSIxIi8+PHJlY3QgeD0iMjA1IiB5PSIxNzQiIGZpbGw9IiNGRkZGRkYiIHdpZHRoPSIxIiBoZWlnaHQ9IjEiLz48cmVjdCB4PSIxNTIiIHk9IjIwMCIgZmlsbD0iI0ZGRkZGRiIgd2lkdGg9IjEiIGhlaWdodD0iMSIvPjxyZWN0IHg9IjUyIiB5PSIyMTEiIGZpbGw9IiNGRkZGRkYiIHdpZHRoPSIyIiBoZWlnaHQ9IjIiLz48cmVjdCB5PSIxOTEiIGZpbGw9IiNGRkZGRkYiIHdpZHRoPSIxIiBoZWlnaHQ9IjEiLz48cmVjdCB4PSIxMTAiIHk9IjE4NCIgZmlsbD0iI0ZGRkZGRiIgd2lkdGg9IjEiIGhlaWdodD0iMSIvPjwvc3ZnPg==);
```
Comment (ppmarkek) [inline src/pages/ErrorPage/styles.ts:44]: Add separate file for the picture

**Review by ppmarkek — APPROVED** [review]

File: src/pages/ErrorPage/ErrorPage.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  return (
+    <Container>
+      <div className="background-img">
+        <Space></Space>
+        <Wrapper>
+          <div className="img-wrapper">
+            <Digit>44</Digit>
+          </div>
+          <Text>
+            The page you are trying to search has been moved to another
```
Comment (1LisaW) [inline src/pages/ErrorPage/ErrorPage.tsx:21]: Could you please add here i18n? Text should be dependand on current language.

File: src/pages/ErrorPage/ErrorPage.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+            universe.
+          </Text>
+          <Button onClick={handleButtonClick}>GET ME HOME</Button>
```
Comment (1LisaW) [inline src/pages/ErrorPage/ErrorPage.tsx:24]: Could you please add here i18n? Text should be dependand on current language.

**Review by 1LisaW — APPROVED** [review]

**Pull request merged**